### DISCUSSION TITLE
Replace the mouse sensitivity slider with a textbox

### DISCUSSION THREAD

**Discussion by TheAirBlow** [ppy/osu#23191] [Ideas] [answered]
It's just annoying as hell to try and get a precise number set. I always get something like 1.37 and my perfectionism is dying, and it doesn't help that I can't make change the decimal point because the sensitivity just doesn't feel right.

**Comment by nagi-desuuu** [comment]
Are the arrow keys not precise enough for you?

**Reply by TheAirBlow** [reply]
> Are the arrow keys not precise enough for you?

Ohh, so that exists. bruh

**Comment by peppy** [comment, marked as answer]
All slider controls will have text input in some future.
